lib/proxy/views.py

```diff
@@ -238,6 +238,26 @@ def sign(self, request):
                            query=urllib.urlencode(qs))
 
 
+def check_sig(request):
+    """
+    Override the check_sig call, Boku doesn't actually implement this,
+    the proxy does because it has access to that data. Rather than
+    send the data on, or try overriding the client, just grab this request,
+    parse it and send back a 204 or 400.
+    """
+    data = request.GET.copy()
+    sig = data.pop('sig')[0]
```

Comment:
won't this raise a KeyError when no ``sig``? Or are you relying on the form to protect against this?